### DISCUSSION TITLE
Mobile app or web app

### DISCUSSION THREAD

**Discussion by edersong** [firefly-iii/firefly-iii#6599] [General]
Is there a plan to develop a mobile app or a web app who will work on mobile phones lika an app??
It's hard o manage my budget within smartphone and apps available doesn't work within http.

**Comment by JC5** [comment]
Firefly III itself works on mobile pretty well. I'm not going to develop another app, there are a lot of those. 

But what do you mean with "http"?

**Reply by edersong** [reply]
Hello, @JC5 
Thank you for the feedback and congratulations to the excellent work there!
The Firefly III works at mobile, sure... but it not fits like a mobile app and the letters and buttons are so small.
About http, I mean that all available apps in Play store, needs that the server works with SSL, I mean, thru https:// and not http:// and, me and some other users that reported in forums, use VPN to access self hosted apps outside, so https:// is not supported.

**Reply by JC5** [reply]
Ah sorry, I can't help you with that.
